### PR TITLE
GCI: Setting Fragment for Server URL

### PR DESCRIPTION
**Pull request by HITGIF**
For the [task](https://codein.withgoogle.com/dashboard/task-instances/5962217176432640/).
- Add settings fragment to nav drawer (can also be triggered from the action in main menu at top right).
- Add setting for setting up/editing server URL for further retrofit calls.
- Subtext displays the current value for the URL.
- Settings are retained as SharedPreferences.

SCREENSHOTS
screenshot_1513592250
screenshot_1513592839
screenshot_1513592183
screenshot_1513592180

### CODE REVIEW

File: app/src/main/java/com/community/jboss/leadmanagement/Contacts/Contact.java [hunk 1]
```diff
@@ -1,53 +0,0 @@
-package com.community.jboss.leadmanagement.Contacts;
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/Contacts/Contact.java]: why removing this file is needed

Comment (HITGIF) [reply app/src/main/java/com/community/jboss/leadmanagement/Contacts/Contact.java]: Oops you have already reverted that commit...
Deleted that again ;)

**Comment by HITGIF** [comment]
Done =)

**Comment by garg-anuj** [comment]
awesome work

**Pull request closed without merging**